doc/onboarding-extras.md

```diff
@@ -2,23 +2,29 @@
 
 | subsystem | maintainers |
 | --- | --- |
-| `lib/buffer` | @trevnorris |
-| `lib/child_process` | @cjihrig, @bnoordhuis |
-| `lib/cluster` | @cjihrig, @bnoordhuis |
+| `lib/assert` | @Trott |
```

Comment:
lib/assert should likely fall under the responsibility of the nodejs/testing team